src/components/settings/General.vue

```diff
@@ -37,6 +37,42 @@
                 </select>
             </div>
 
+            <!-- Time Format -->
+            <div class="mb-4">
+                <label class="form-label">
+                    {{ $t("Time Format") }}
+                </label>
+
+                <div class="form-check">
+                    <input
+                        id="timeFormatIndex12Hour"
+                        v-model="settings.use12HourTimeFormat"
+                        class="form-check-input"
+                        type="radio"
+                        name="timeFormatIndex"
+                        :value="true"
+                        required
+                    />
+                    <label class="form-check-label" for="timeFormatIndex12Hour">
+                        {{ $t("12-hour") }}
```

Comment:
Please add all translatable strings to `en.json`, otherwise our translators will not be able to pick this up